apps/desktop/desktop/src/main/java/bisq/desktop/main/content/authorized_role/mediator/MediatorController.java

```diff
@@ -244,7 +244,13 @@ private void update() {
 
     private void maybeSelectFirst() {
         if (!model.getListItems().getFilteredList().isEmpty()) {
-            UIThread.runOnNextRenderFrame(() -> selectionService.selectChannel(model.getListItems().getSortedList().get(0).getChannel()));
+            UIThread.runOnNextRenderFrame(() -> {
+                if (model.getListItems().getFilteredList().isEmpty()) {
+                    log.warn("UI thread check found no mediation cases, skipping");
+                } else {
+                    selectionService.selectChannel(model.getListItems().getSortedList().getFirst().getChannel());
```

Comment:
We stick with java 17 language feature, `getFirst()` is not supported in Java 17.